index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>CV</title>
+  <link href="https://fonts.googleapis.com/css?family=Changa|Monda:400,700" rel="stylesheet">
+  <link href="css/normalize.css" rel="stylesheet">
+  <link href="css/reset.css" rel="stylesheet">
+  <link href="css/style.css" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <figure class="avatar">
+      <img src="img/myphoto.jpg" alt="I'm" height="200">  
+    </figure>
+    <h1>Aliaksei Barysionak<br>
```

Comment:
I prefer don't use <br>. Check https://stackoverflow.com/questions/7524185/can-we-add-a-span-inside-h1-tag, please, - looks like we can use span inside h1.